app/src/main/java/de/grobox/transportr/utils/DateUtils.java

```diff
@@ -83,13 +83,7 @@ public static String getDelayString(long delay) {
 
 	@Nullable
 	public static String getDelayText(long delay) {
-		if (delay >= 0) {
-			return "+" + Long.toString(delay / 1000 / 60);
-		} else if (delay < 0) {
-			return "-" + Long.toString(delay / 1000 / 60);
-		} else {
-			return null;
-		}
+		return (delay >= 0 ? "+" : "") + Long.toString(delay / 1000 / 60);
```

Comment:
I think this isn't `@Nullable` anymore, right? Could you please remove the annotation and check all usages of this function please and remove `null` handling there?

Reply:
True, I have removed the annotation, but the only two usages were not handling the `null` case anyway (as I have been changing them within this PR).